{"name": "test_data_pushing_standard_fields_contact", "status": "skipped", "statusDetails": {"message": "Skipped: Skipping test due to unexpected login error: InvalidSessionIdException", "trace": "('C:\\\\Users\\\\usmaan.arshad\\\\PycharmProjects\\\\PythonProject\\\\tests\\\\Push_Data_to_Standard_Fields\\\\test_data_pushing_standard_fields_contact.py', 77, 'Skipped: Skipping test due to unexpected login error: InvalidSessionIdException')"}, "start": 1745405398732, "stop": 1745405410535, "uuid": "ecb2540d-5008-4d88-92a0-78b15f549e93", "historyId": "1b1188d3f9fbef01e757e879ffa93f68", "testCaseId": "1b1188d3f9fbef01e757e879ffa93f68", "fullName": "tests.Push_Data_to_Standard_Fields.test_data_pushing_standard_fields_contact#test_data_pushing_standard_fields_contact", "labels": [{"name": "feature", "value": "Mapping - Contact field Mapping"}, {"name": "severity", "value": "critical"}, {"name": "story", "value": "Validate successful mapping of contact fields."}, {"name": "tag", "value": "Skipped"}, {"name": "parentSuite", "value": "tests.Push_Data_to_Standard_Fields"}, {"name": "suite", "value": "test_data_pushing_standard_fields_contact"}, {"name": "host", "value": "RT-LT-597"}, {"name": "thread", "value": "9800-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.Push_Data_to_Standard_Fields.test_data_pushing_standard_fields_contact"}]}